### PR TITLE
XMDS: add documentation for 4.1 alerts, weather.

### PR DESCRIPTION
**Pull request by dasgarner**

### CODE REVIEW

File: docs/creating-a-player/xmds.md [hunk 1]
```diff
@@ -621,7 +666,41 @@ or it can provide more information
...
+**Since CMS 4.1 and later** displays have been able to submit alerts to the CMS via the SubmitLog method. Alerts represent interesting events occurring on the media player, 
+such as the network going offline, application restarts, memory alerts, etc.
+
+These are submitted by setting the category attribute to `category="alert"`.
 
+The following format is expected for alerts:
+
+```xml
+<logs>
+    <log date="Y-m-d H:i:s" category="alert">
```
Comment (PeterMis) [inline docs/creating-a-player/xmds.md:679]: the category is `event` not `alert` - although to be fair alert might make more sense, but we would need to change the CMS code.

File: docs/creating-a-player/xmds.md [hunk 2]
```diff
@@ -621,7 +666,41 @@ or it can provide more information
...
+        <eventType>eventType</eventType>
+        <alertType>alertType</alertType>
+        <refId>refId</refId>
+        <message>message</message>
+    </log>
+</logs>
+```
+
+ - date: The local date, ISO formatted
+ - category: `alert`
```
Comment (PeterMis) [inline docs/creating-a-player/xmds.md:689]: as above

**Review by PeterMis — COMMENTED** [review]
Looks good, just a small comment on the alert/event log xml

File: docs/creating-a-player/xmds.md (hunk 1 above)
Comment (dasgarner) [reply docs/creating-a-player/xmds.md:679]: Ah OK, I'll change it.

**Pull request merged**